analyzers/its/expected/CSharpLatest/S6966-NetCore31.MVC.ConfigurableRules-netcoreapp3.1.Views.json

```diff
@@ -4,7 +4,7 @@
       "Id": "S6966",
       "Message": "Await RenderSectionAsync instead.",
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/CSharpLatest/NetCore31WithConfigurableRules/Views/Shared/_Layout.cshtml#L46",
```

Comment:
[Link](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/CSharpLatest/NetCore31WithConfigurableRules/Views/Shared/_Layout.cshtml#L46)
This change shifts the reported issue on `@RenderSection("Scripts", required: false)` 1 char before. Seems related to changes in Razor location mapping, that come with the new version of the .NET SDK.

Reply:
The problem only appears on my machine actually. I have reverted the commit and have the CI passing, without this change. Most likely connected by the different behavior that different versions of Roslyn have, w.r.t. mapping.
Resolving.